Gemfile

```diff
@@ -247,6 +247,8 @@ group :development, :test do
   # for `rake doc:liquid:generate` and similar
   gem 'source2swagger', git: 'https://github.com/3scale/source2swagger'
   gem 'unicorn-rails'
+
+  gem 'rswag'
```

Comment:
You are not going to need this anymore. The library is to generate the jsons from the code, but we decided to leave that for another [issue ](https://issues.redhat.com/browse/THREESCALE-8904)